README.md

```diff
@@ -11,7 +11,7 @@ let Statsum = require('statsum');
 // Create a client object
 let configurer = async (project) => { return {
   project: 'example-project',
-  baseUrl: 'https://example.com',
+  rootUrl: 'https://example.com',
```

Comment:
This is not a TC thing... :)

Any place we use `node-statsum` we learn the `baseUrl` of the statsum server from `auth.statsumToken` which contains the `baseUrl`:
https://docs.taskcluster.net/reference/platform/taskcluster-auth/references/api#get-token-for-statsum-project

Reply:
It doesn't always come from `auth.statsumToken`! Not in auth itself anyway.